plugins/auth-backend-module-aws-alb-provider/src/authenticator.ts

```diff
@@ -81,8 +81,12 @@ export const awsAlbAuthenticator = createProxyAuthenticator({
       return {
         result: {
           fullProfile,
+          accessToken: accessToken,
+          expiresInSeconds: claims.exp,
```

Comment:
As noted elsewhere, this is not `expiresInSeconds` but instead `expiresAt`. No action on that right now as we're currently only looking to restore the original behavior.

Reply:
Actually the original behavior also uses `expiresInSeconds` but it is actually value of `expiresAt` 😄

Reply:
yep that's what I meant 😅, we're keeping that original behavior intact for now